File: repos_estudiantes_ALVAREZ_primera_entrega/ILLANES_VASQUEZ_GABRIEL/IntroduccionAPOO/EJERCICIO1/Persona.java

package ejercicio7;

public class Persona {


	
	public String nombre;
	public String ciudad;
	public int edad;
	
	public Persona (String nombre,String ciudad, int edad ) {
		this.ciudad=ciudad;
		this.edad=edad;
		this.nombre=nombre;
	}
	
	public void mostrarSaludo() {
		System.out.println("Hola mi nombre es: " +nombre+ " de la ciudad de :"+ciudad );
	}

	public void mostrarEdad(){
		
		if (this.edad>18) {
			System.out.println("la persona:" +this.nombre + " es mayor de edad"); 
			
		}else {
			System.out.println("la persona:" +this.nombre + "  es menor de edad de edad");
		}
	}

	public static void main(String[] args) {
		Persona p1 = new Persona("Javier", "Lapaz",12);
		Persona p2 = new Persona("Juan", "SantaCruz", 20);
		Persona p3 = new Persona("Eduardo", "Lapaz",14);
		
		p1.mostrarSaludo();
		p2.mostrarSaludo();
		p3.mostrarSaludo();

		p1.mostrarEdad();
		p2.mostrarEdad();
		p3.mostrarEdad();
		
	}

}

File: repos_estudiantes_ALVAREZ_primera_entrega/ILLANES_VASQUEZ_GABRIEL/IntroduccionAPOO/EJERCICIO1/Persona.py
class Persona:

    def __init__(self, nombre, ciudad, edad):
        self.nombre = nombre
        self.ciudad = ciudad
        self.edad = edad

    def mostrarSaludo(self):
        print("Hola mi nombre es:", self.nombre, "de la ciudad de:", self.ciudad)

    def mostrarEdad(self):
        if self.edad > 18:
            print("la persona:", self.nombre, "es mayor de edad")
        else:
            print("la persona:", self.nombre, "es menor de edad")

if __name__ == "__main__":
    p1 = Persona("Javier", "Lapaz", 12)
    p2 = Persona("Juan", "SantaCruz", 20)
    p3 = Persona("Eduardo", "Lapaz", 14)

    p1.mostrarSaludo()
    p2.mostrarSaludo()
    p3.mostrarSaludo()

    p1.mostrarEdad()
    p2.mostrarEdad()
    p3.mostrarEdad()
